addons/uh60_mfd/functions/fnc_perSecond.sqf

```diff
@@ -8,6 +8,8 @@
 
 params ["_vehicle"];
 
+[_vehicle] call vtx_uh60_mfd_fnc_storePylons;
```

Comment:
Maybe should be done in `changeBrightness` and `configureHMD` instead of perSecond?

Reply:
The intent is to store the configuration halfway often to manage cases like being reloaded by an ammo vehicle, etc. where the pilot won't be getting out of the aircraft.

I wanted to also catch the case where one or more of the EICAS messages could be asserted and it would be restored correctly.

In local testing this seemed to mitigate the "HMD fluid" issue.

Reply:
We want to store the intentional changes, which as far as I can see are done by the brightness interactions and the color-from-settings. What other changes do we expect to need to store?

Reply:
I wanted to pickup the EICAS warnings as well which could assert or de-assert independent of an HMD action: 
https://docs.google.com/spreadsheets/d/1FUSzAR3CNno2-H-3twC4JH8l7Z7tsOUtHhMDkFxVoCM/edit#gid=0

Example: we could have a chip light come on due to damage. Pilot flies back for rearm (repair is unavailable) without interacting with the HMD. The rearm action would "hide" the fact that we have a chip light on if we don't store that information at some frequency.

Reply:
I see, yes, for everything that might go in the pylons. That speaks more to using a proper setter function to change any pylon value. PerSecond doesn't run when no pilot is in, so if they get out to repair and rearm, the stored values would be out of date.

Reply:
Ah, good point. I hadn't thought of instances where the pilot leaves the aircraft. 

I can come up with a function to update the pylon value and then store it, then work it into the existing pylon usage. It will take me a little while.

Thanks

Reply:
Initially addressed in commit: https://github.com/Project-Hatchet/H-60/pull/247/commits/ac5ca91f2bb031ef2ea8cec5328b69d3d29f8f38